collect_app/build.gradle

```diff
@@ -282,6 +285,16 @@ dependencies {
     androidTestImplementation group: 'com.squareup.okhttp3', name: 'mockwebserver', version: '3.9.0'
 }
 
+// https://stackoverflow.com/questions/37347326/errorconflict-with-dependency-com-google-code-findbugsjsr305#
```

Comment:
Is this an issue because of the screengrab dependency? That should also be documented here.

Reply:
This issue does not occur due to screengrab dependency. During the instrumentation test run, if the main APK and the test APK use the same library but in different versions, then gradle build fails.  I added this just to make sure both APKs use the same version.

Reply:
Can you please submit this as a separate PR if it's not related to this feature?

Reply:
I have added a warning in opendatakit/docs#487 to avoid this issue https://github.com/opendatakit/docs/pull/487/files#diff-6b91fcfb30169cf52427f8f9ce3a77b6R208.
So, I  think this is no more needed.